### PR TITLE
Update ExtensionDownloader.ps1 - bug fixes

### PR DESCRIPTION
**Pull request by kmyrup**
Update ExtensionDownloader.ps1 to fix a few small things.

### CODE REVIEW

File: tools/ExtensionDownloader.ps1 [hunk 1]
```diff
@@ -134,15 +134,13 @@ function Main
             Write-Host "##vso[task.setvariable variable=codeSignPolicyFile;]$pFile"
             Write-Host "##vso[task.setvariable variable=codeSignEnabled]true"
             $itemDir = $extensionRootDirectory + "\objects\consoleextension\" + $objectName;
-            $cabFile = $itemDir + "\" + $objectName + ".cab"
+            $cabFile = $itemDir + "\" + $objectName
```
Comment (silveyy) [inline tools/ExtensionDownloader.ps1:137]: >$objectName [](start = 40, length = 11)

why this change?

File: tools/ExtensionDownloader.ps1 [hunk 2]
```diff
@@ -134,15 +134,13 @@ function Main
...
 
             $r = mkdir $itemDir;
     
-            if((Test-Path $cabFile) -eq $False )
```
Comment (kmyrup) [inline tools/ExtensionDownloader.ps1:141]: > if((Test-Path $cabFile) -eq $False ) [](start = 11, length = 37)

If already exists, write-error instead.

File: tools/ExtensionDownloader.ps1 [hunk 3]
```diff
@@ -203,8 +201,9 @@ function print-Summary
 # ===================================================================
 function print-EnvironmentVariables
 {
-    write-host "Environment variables:";
-    get-childitem env:
+    # Enable if needed for debugging
+    #write-host "Environment variables:";
+    #get-childitem env:
```
Comment (kmyrup) [inline tools/ExtensionDownloader.ps1:206]: will put this back (for now)

File: tools/ExtensionDownloader.ps1 [hunk 4]
```diff
@@ -138,11 +138,15 @@ function Main
 
             $r = mkdir $itemDir;
     
-            if((Test-Path $cabFile) -eq $False )
+            # Always download to ensure we are verifying the correct latest file
+            if ((Test-Path $cabFile) -eq $True)
             {
-                Write-Host "Downloading cab:" $objectInfo.downloadLocation "to:" $itemDir;
-                Invoke-WebRequest -Uri $objectInfo.downloadLocation -OutFile $cabFile;
+                Write-Error "File: " $itemDir + " already exists. This is unexpected.";
```
Comment (silveyy) [inline tools/ExtensionDownloader.ps1:144]: > +  [](start = 45, length = 3)

1. don't need the + sign... it will actually just get logged in the output.
2. you should log the $cabFile, not the directory.
3. you don't need the space after the ':' or before the " already" (see line 148 below)

**Review by silveyy — APPROVED** [review]
:shipit:
